gnat2goto/driver/tree_walk.adb

```diff
@@ -4094,7 +4097,21 @@ package body Tree_Walk is
       Global_Symbol_Table.Replace (Proc_Name, Proc_Symbol);
    end Do_Subprogram_Body;
 
-   -------------------------------
+   -----------------------------
+   -- Do_Subprogram_Body_Stub --
+   -----------------------------
+
+   procedure Do_Subprogram_Body_Stub (N : Node_Id) is
+   begin
+      --  The Gnat compilation model requires that a file containing the
+      --  which is the separate subprogram body is present otherwise
+      --  a compilation error is generated.
```

Comment:
I'm not sure I understand this sentence.

Reply:
Corrected comment